src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift

```diff
@@ -11,15 +11,17 @@ class AppInformationViewController: DynamicTableViewController, NavigationBarOpa
 	
 	init(
 		elsService: ErrorLogSubmissionProviding,
+		store: Store,
```

Comment:
Do we have a better option rather an injecting store here and passing everywhere?

Reply:
Unfortunately, I didn't find the better option!